packages/base-styles/index.js

```diff
@@ -0,0 +1,60 @@
+exports.adminColorSchemes = {
```

Comment:
"theme" vs"color scheme"?

Reply:
I guess it is, I shared the link to the WordPress docs where they refer to it as `admin color schemes`.